gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -76,7 +75,7 @@
   static final int MAX_POOL_SIZE = 1000;
 
   private final int processorCount;
-  private final ExecutorProvider executorProvider;
+  private final ExecutorService executor;
```

Comment:
Sorry for the incremental reviews, I didnt notice earlier. Why ExecutorService over a plain executor?

Reply:
Fixed. I'd originally thought to use the abstraction above ScheduledExecutorService, but Executor works too.